src/resultsbottomuppage.cpp

```diff
@@ -95,6 +95,33 @@ ResultsBottomUpPage::ResultsBottomUpPage(FilterAndZoomStack* filterStack, PerfPa
                 }
             }
         });
+
+    connect(parser, &PerfParser::bottomUpDataPerThreadAvailable, this,
+            [this](const Data::BottomUpResults& data) { bottomUpByThread = data; });
+
+    connect(parser, &PerfParser::bottomUpDataPerProcessAvailable, this,
+            [this](const Data::BottomUpResults& data) { bottomUpByProcess = data; });
+
+    connect(parser, &PerfParser::bottomUpDataPerCpuAvailable, this,
+            [this](const Data::BottomUpResults& data) { bottomUpByCpu = data; });
+
+    for (const auto& aggregationType :
+         {QLatin1String("Symbol"), QLatin1String("Thread"), QLatin1String("Process"), QLatin1String("CPU")}) {
```

Comment:
see below, make it an enum and add a mapping from enum to translated string here